signal_scanner_bot/signal.py

```diff
@@ -45,6 +44,45 @@ def message_timestamp(data: Dict) -> datetime:
     return dt
 
 
+def list_identities() -> str:
+    """
+        Function that calls the signal-cli `listIdentities` command and returns the entire result as a string
+    """
+    with env.SIGNAL_LOCK:
+        proc = subprocess.run(
+            [
+                "signal-cli",
+                "-u",
+                str(env.BOT_NUMBER),
+                "listIdentities"
+            ],
+            capture_output=True,
+        )
+    if proc.stdout:
+        return proc.stdout.decode('utf-8')
+    if proc.stderr:
+        log.warning(f"STDERR: {proc.stderr.decode('utf-8')}")
+
+
+def trust_identity(phone_number: str, safety_number: str):
+    """
+        Function that calls the signal-cli `trust` command for the provided phone + safety numbers
+    """
+    with env.SIGNAL_LOCK:
+        proc = subprocess.run(
+            [
+                "signal-cli",
+                "-u",
+                str(env.BOT_NUMBER),
+                "trust",
+                phone_number,
+                "-v",
+                "\"" + safety_number + "\""
```

Comment:
This might be simplified:
```suggestion
                f'"{safety_number}"'
```